src/components/searchForm.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import InputWithAutocomplete from './inputWithAutocomplete';
+
+class SearchForm extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            inputText: '',
+            isSubmitable: false
+        }
+
+        this.handleChange = this.handleChange.bind(this);
+    }
+
+    handleChange(value) {
+        this.setState({
+            inputText: value,
+            isSubmitable: this.isValid(value)
+        });
+    }
+
+    isValid(part) {
```

Comment:
Slice is fuckup (its mostly copypasted expression from input component) 
Cant use includes to find string in the array with objects so filter. We discussed that I will create another array with only names, so be it